src/plots/cartesian/layout_attributes.js

```diff
@@ -270,7 +270,7 @@ module.exports = {
             ],
             editType: 'calc',
             description: [
-                'Sets the lower and upper bounds of this axis break.',
+                'Sets the lower and upper bounds of this axis rangebreak.',
```

Comment:
hmm somehow as a noun I'm ambivalent about it - probably I'd change the noun usages in descriptions back to break(s) but it's not as clear to me as when break is a verb...